<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border-radius: 2px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_title">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}

QPushButton:hover{
background-color: rgb(255, 0, 0);
}

QLabel{
	color: rgb(0, 0, 0);
	font: 63 16pt &quot;Yu Gothic UI Semibold&quot;;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>276</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="bt_minimize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}

QPushButton:hover{
background-color: rgb(200, 200, 200);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/minus.svg</normaloff>imagenes/minus.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_maximize">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}

QPushButton:hover{
background-color: rgb(200, 200, 200);
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/chevron-up.svg</normaloff>imagenes/chevron-up.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_normal">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
}

QPushButton:hover{
background-color: rgb(200, 200, 200);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/chevron-down.svg</normaloff>imagenes/chevron-down.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_close">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>imagenes/x.svg</normaloff>imagenes/x.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_middle">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>5</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color: rgb(255, 85, 0);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_content">
         <property name="styleSheet">
          <string notr="true">QLabel{
background-color: rgb(255, 255, 255);
font: 75 18pt &quot;Leelawadee UI&quot;;
color: rgb(0, 0, 0);
}

QPushButton{
background-color: rgb(255, 255, 255);
border-radius: 15px;
border: 1px solid rgb(255, 85, 0);
	font: 12pt &quot;Segoe UI Historic&quot;;
	color: rgb(255, 85, 0);
}

QPushButton:hover{
background-color: rgb(255, 85, 0);
border-radius: 15px;
border: 1px solid rgb(255, 85, 0);
	font: 12pt &quot;Segoe UI Historic&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:pressed{
background-color: rgb(230, 76, 0);
border-radius: 15px;
border: 1px solid rgb(255, 85, 0);
	font: 12pt &quot;Segoe UI Historic&quot;;
	color: rgb(255, 255, 255);
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,1">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>9</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,10,1,12">
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>40</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0">
                <property name="spacing">
                 <number>1</number>
                </property>
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <property name="rightMargin">
                 <number>1</number>
                </property>
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,0,1">
                  <item>
                   <widget class="QPushButton" name="bt_logout">
                    <property name="minimumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton{
background-color: rgb(255, 255, 255);
border-radius: 15px;
border: 1px solid rgb(0, 170, 255);
	font: 12pt &quot;Segoe UI Historic&quot;;
	color: rgb(0, 170, 255);
}

QPushButton:hover{
background-color: rgb(0, 170, 255);
border-radius: 15px;
border: 1px solid rgb(0, 170, 255);
	font: 12pt &quot;Segoe UI Historic&quot;;
	color: rgb(255, 255, 255);
}

QPushButton:pressed{
background-color: rgb(0, 127, 191);
border-radius: 15px;
border: 1px solid rgb(0, 127, 191);
	font: 12pt &quot;Segoe UI Historic&quot;;
	color: rgb(255, 255, 255);
}</string>
                    </property>
                    <property name="text">
                     <string>Logout</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="bt_go_to_database">
                    <property name="minimumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>120</width>
                      <height>30</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="text">
                     <string>Go to Database</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="PlotWidget" name="graphWidget" native="true">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>200</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>13</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="frame_3">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1,15">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>1</number>
                </property>
                <property name="rightMargin">
                 <number>1</number>
                </property>
                <property name="bottomMargin">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_5">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1">
                   <property name="spacing">
                    <number>1</number>
                   </property>
                   <property name="leftMargin">
                    <number>20</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <item>
                    <spacer name="verticalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>67</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bt_screenshot">
                     <property name="minimumSize">
                      <size>
                       <width>150</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>150</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QPushButton{
background-color: rgb(120,120,120);
font: 10pt &quot;Lucida Fax&quot;;
color: white;
border-radius: 15px;
border: None;
}

QPushButton:hover{
background-color: rgb(66, 66, 66);
border-radius: 15px;
font: 10pt &quot;Lucida Fax&quot;;
color: white;
border: None;
}

QPushButton:pressed{
background-color: rgb(255,255,255);
font: 10pt &quot;Lucida Fax&quot;;
color: black;
border-radius: 15px;
border :None;
}</string>
                     </property>
                     <property name="text">
                      <string>Graph's screenshot</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_5">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>67</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>24</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QFrame" name="frame_4">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,1,1,1">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
                     <item>
                      <widget class="QDial" name="graph1">
                       <property name="cursor">
                        <cursorShape>UpArrowCursor</cursorShape>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLCDNumber" name="Num1"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
                     <item>
                      <widget class="QDial" name="graph2">
                       <property name="cursor">
                        <cursorShape>UpArrowCursor</cursorShape>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLCDNumber" name="Num2"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
                     <item>
                      <widget class="QDial" name="graph3">
                       <property name="cursor">
                        <cursorShape>UpArrowCursor</cursorShape>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLCDNumber" name="Num3"/>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
                     <item>
                      <widget class="QDial" name="graph4">
                       <property name="cursor">
                        <cursorShape>UpArrowCursor</cursorShape>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLCDNumber" name="Num4"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>8</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
